### PR TITLE
vegeta: add French translation

### PR DESCRIPTION
**Pull request by melvinxeu**
- [X The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

File: pages.fr/common/vegeta.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# vegeta
+
+> Un utilitaire de ligne de commande et une bibliothèque pour les tests de charge HTTP.
+> Voir aussi `ab`.
+> Plus d'informations: <https://github.com/tsenart/vegeta>.
+
+- Lancer une attaque d'une durée de 30 secondes:
```
Comment (navarroaxel) [inline pages.fr/common/vegeta.md:7]:
```suggestion
- Lancer une attaque d'une durée de 30 secondes :
```

File: pages.fr/common/vegeta.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# vegeta
+
+> Un utilitaire de ligne de commande et une bibliothèque pour les tests de charge HTTP.
+> Voir aussi `ab`.
+> Plus d'informations: <https://github.com/tsenart/vegeta>.
```
Comment (navarroaxel) [inline pages.fr/common/vegeta.md:5]:
```suggestion
> Plus d'informations : <https://github.com/tsenart/vegeta>.
```

File: pages.fr/common/vegeta.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- Lancer une attaque d'une durée de 30 secondes:
+
+`echo "{{GET https://exemple.com}}" | vegeta attack -duration={{30s}}`
+
+- Lancez une attaque sur un serveur avec un certificat HTTPS auto-signé.:
```
Comment (navarroaxel) [inline pages.fr/common/vegeta.md:11]:
```suggestion
- Lancez une attaque sur un serveur avec un certificat HTTPS auto-signé :
```

File: pages.fr/common/vegeta.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Lancer une attaque avec un taux de 10 demandes par seconde:
+
+`echo "{{GET https://exemple.com}}" | vegeta attack -duration={{30s}} -rate={{10}}`
+
+- Lancer une attaque et afficher un rapport:
+
+`echo "{{GET https://exemple.com}}" | vegeta attack -duration={{30s}} | vegeta report`
+
+- Lancer une attaque et reporter les résultats sur un graphique (latence en fonction du temps):
```
Comment (navarroaxel) [inline pages.fr/common/vegeta.md:23]:
```suggestion
- Lancer une attaque et reporter les résultats sur un graphique (latence en fonction du temps) :
```

File: pages.fr/common/vegeta.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+- Lancez une attaque sur un serveur avec un certificat HTTPS auto-signé.:
+
+`echo "{{GET https://exemple.com}}" | vegeta attack -insecure -duration={{30s}}`
+
+- Lancer une attaque avec un taux de 10 demandes par seconde:
+
+`echo "{{GET https://exemple.com}}" | vegeta attack -duration={{30s}} -rate={{10}}`
+
+- Lancer une attaque et afficher un rapport:
```
Comment (navarroaxel) [inline pages.fr/common/vegeta.md:19]:
```suggestion
- Lancer une attaque et afficher un rapport :
```

File: pages.fr/common/vegeta.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+- Lancer une attaque d'une durée de 30 secondes:
+
+`echo "{{GET https://exemple.com}}" | vegeta attack -duration={{30s}}`
+
+- Lancez une attaque sur un serveur avec un certificat HTTPS auto-signé.:
+
+`echo "{{GET https://exemple.com}}" | vegeta attack -insecure -duration={{30s}}`
+
+- Lancer une attaque avec un taux de 10 demandes par seconde:
```
Comment (navarroaxel) [inline pages.fr/common/vegeta.md:15]:
```suggestion
- Lancer une attaque avec un taux de 10 demandes par seconde :
```

File: pages.fr/common/vegeta.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+- Lancer une attaque et afficher un rapport:
+
+`echo "{{GET https://exemple.com}}" | vegeta attack -duration={{30s}} | vegeta report`
+
+- Lancer une attaque et reporter les résultats sur un graphique (latence en fonction du temps):
+
+`echo "{{GET https://exemple.com}}" | vegeta attack -duration={{30s}} | vegeta plot > {{chemin/au/results.html}}`
+
+- Lancer une attaque contre plusieurs URL à partir d'un fichier:
```
Comment (navarroaxel) [inline pages.fr/common/vegeta.md:27]:
```suggestion
- Lancer une attaque contre plusieurs URL à partir d'un fichier :
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**